docs/traceability/index.md

```diff
@@ -262,7 +262,7 @@ apic_traceability_agent:
 output.traceability:
   enabled: true
   hosts:
-   - ${TRACEABILITY_HOST:"ingestion-lumberjack.datasearch.axway.com:453"}
+   - ${TRACEABILITY_HOST:"ingestion.datasearch.axway.com:5044"}
```

Comment:
I hope we did a test with publishing events to ingestion.datasearch.axway.com:5044

Reply:
I was able to view published events in a production env.

Reply:
Perfect, thanks :)